requirements.txt

```diff
@@ -0,0 +1,3 @@
+django
+django-crispy-forms
```

Comment:
To automatycznie ściąga zależności? Czy po prostu, żeby łatwiej trackować

Reply:
żeby łatwiej było ogarnąć, jak są wypisane w ten sposób to wtedy wystarczy dowalić `pip install -r requirements.txt`